### DISCUSSION TITLE
Remote modification while SMB share is mounted

### DISCUSSION THREAD

**Discussion by Zulgrib** [dokan-dev/dokany#1124] [Q&A]
Hello,
What should officially happen if a computer A mounts an SMB share using mirror.exe but a file is added, modified, removed, renamed, have it's ACL altered by a machine B… will the mountpoint on computer A reflect the remote modifications or will we have issues ?

Does it still apply if computer B mounted the SMB share using mirror.exe too instead of being the server doing the share itself or if it was mounted/accessed using Microsoft's tooling ?

**Comment by Liryna** [comment]
Hi @Zulgrib ,

I am not sure I correctly understood the question but here is some information:
- Mirror does not detect and notify remote changes automatically. It would need to use the `Notify*` API for that https://github.com/dokan-dev/dokany/blob/master/dokan/dokan.h#L1003-L1021
- If a file is manipulated after the ACL change  (for example if write permission were removed), the write request will fail when the remote tries to apply it and SMB server will send the failure back to the client.
